postcode_lookup/template_sorter.py

```diff
@@ -151,9 +151,12 @@ def context(self):
         context["can_register_vac"] = self.timetable.is_before(
             TimetableEvent.VAC_APPLICATION_DEADLINE
         )
-        context["htag"] = "h2"
+        context["htag_primary"] = "h2"
+        context["htag_secondary"] = "h3"
         if self.response_type == ResponseTypes.MULTIPLE_DATES:
-            context["htag"] = "h3"
+            context["htag_primary"] = "h3"
+            context["htag_secondary"] = "h4"
```

Comment:
This was a little issue I fixed while I was here.
Previously if `htag` was `h3` we'd render the next heading as a `h3` too